config/autoload/local.php.dist

```diff
@@ -1,3 +1,9 @@
 <?php
+
+use Zend\ConfigAggregator\ConfigAggregator;
+
 return [
+    // Cache the configuration: Recommended for production
```

Comment:
Wording could be improved:
> Recommended to be enabled for production
I don't think it is clear enough that this is the development config.